### DISCUSSION TITLE
Mouse lag

### DISCUSSION THREAD

**Discussion by S1lverone** [Lailloken/Exile-UI#449] [Q&A]
Hello.
first i wanna start telling how clever and fantastic this script is. I'm amazed about the features especially the clone screen which I mainly use it for.
Can anyone help me understand why my mouse is lagging when I use it. The mouse Spike jump around the screen, makes this tool unuseable to me. 
cheers

**Comment by Lailloken** [comment]
Could you check the size of your client.txt log-file that's inside `Path of Exile\logs`?
It's possible that this file has become bloated over time. If it's indeed large, close the game and script, and then rename the file to keep it as backup. Then start the game so that it creates a new file.

EDIT: before starting the tool again, travel between a few random waypoints, so the new log-file is not so empty.

To make the tool track your active character again, you'll have to use the in-game `/whois XYZ` command (XYZ being your character's name), or level up again.

**Comment by S1lverone** [comment]
[image: image.png]

is this a big size?

On Fri, 9 Aug 2024 at 18:02, Lailloken ***@***.***> wrote:

> Could you check the size of your client.txt log-file that's inside Path
> of Exile\logs?
> It's possible that this file has become bloated over time. If it's indeed
> large, close the game and script, and then rename the file to keep it as
> backup. Then start the game so that it creates a new file.
>
> To make the tool track your active character again, you'll have to use the
> in-game /whois XYZ command (XYZ being your character's name), or level up
> again.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Lailloken/Lailloken-UI/discussions/449#discussioncomment-10289055>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIY55UBECII4MR7JTF5VE63ZQTR2HAVCNFSM6AAAAABMIW54R6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRYHEYDKNI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Med klimavenlig hilsen

John Aalegaard Larsen

tlf. +45 52 40 50 60



*Denne e-mail kan indeholde fortrolig information. Hvis du ved en
fejltagelse modtager e-mailen, beder vi dig ikke læse indholdet og venligst
informere John Aalegaard med det samme samt slette e-mailen. If you are not
the addressee of this e-mail and/or attachments, please inform John
Aalegaard immediately and delete this e-mail and/or attachments.

**Comment by S1lverone** [comment]
Hey m8.
i have been testing it now after deleting the Client.txt. it did not make
proper solution to the cause. The mouse still have it's moments where it
freezes for a second and not register mouse movement.
is there anything else I can try to do to solve this matter?

On Fri, 9 Aug 2024 at 23:36, john Aalegaard Larsen ***@***.***>
wrote:

> [image: image.png]
>
> is this a big size?
>
> On Fri, 9 Aug 2024 at 18:02, Lailloken ***@***.***> wrote:
>
>> Could you check the size of your client.txt log-file that's inside Path
>> of Exile\logs?
>> It's possible that this file has become bloated over time. If it's indeed
>> large, close the game and script, and then rename the file to keep it as
>> backup. Then start the game so that it creates a new file.
>>
>> To make the tool track your active character again, you'll have to use
>> the in-game /whois XYZ command (XYZ being your character's name), or
>> level up again.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Lailloken/Lailloken-UI/discussions/449#discussioncomment-10289055>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AIY55UBECII4MR7JTF5VE63ZQTR2HAVCNFSM6AAAAABMIW54R6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRYHEYDKNI>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***
>> com>
>>
>
>
> --
> Med klimavenlig hilsen
>
> John Aalegaard Larsen
>
> tlf. +45 52 40 50 60
>
>
>
> *Denne e-mail kan indeholde fortrolig information. Hvis du ved en
> fejltagelse modtager e-mailen, beder vi dig ikke læse indholdet og venligst
> informere John Aalegaard med det samme samt slette e-mailen. If you are not
> the addressee of this e-mail and/or attachments, please inform John
> Aalegaard immediately and delete this e-mail and/or attachments.
>


-- 
Med klimavenlig hilsen

John Aalegaard Larsen

tlf. +45 52 40 50 60



*Denne e-mail kan indeholde fortrolig information. Hvis du ved en
fejltagelse modtager e-mailen, beder vi dig ikke læse indholdet og venligst
informere John Aalegaard med det samme samt slette e-mailen. If you are not
the addressee of this e-mail and/or attachments, please inform John
Aalegaard immediately and delete this e-mail and/or attachments.

**Reply by Lailloken** [reply]
In the "screen-checks" section of the settings, there's an option `enable background pixel-checks`. You could try disabling that to see if it makes a difference.

This might sound silly, but you could also try unplugging you keyboard to see if your mouse-movement improves without a keyboard.

**Comment by S1lverone** [comment]
disable background pixel-checks helps. also closing my app software called
WindowsTop. Now when that is disabled the function for item check and map
checking is not working. is there a work around for this?
out of curiosity, how did you manage to make an AHK script search so fast
for the text it see in the game. it is almost close to instant. very
impressive. My community have been working for a similar tool to diablo 4,
but it has a second delay before it evaluates items text from images. which
make it slower than our self. so of nu Super use. :)
cheers


On Sat, 10 Aug 2024 at 07:17, Lailloken ***@***.***> wrote:

> In the "screen-checks" section of the settings, there's an option enable
> background pixel-checks. You could try disabling that to see if it makes
> a difference.
>
> This might sound silly, but you could also try unplugging you keyboard to
> see if your mouse-movement improves without a keyboard.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Lailloken/Lailloken-UI/discussions/449#discussioncomment-10293935>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIY55UCJDEDSVGT7Y2XF5WTZQWO6NAVCNFSM6AAAAABMIW54R6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRZGM4TGNI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Med klimavenlig hilsen

John Aalegaard Larsen

tlf. +45 52 40 50 60



*Denne e-mail kan indeholde fortrolig information. Hvis du ved en
fejltagelse modtager e-mailen, beder vi dig ikke læse indholdet og venligst
informere John Aalegaard med det samme samt slette e-mailen. If you are not
the addressee of this e-mail and/or attachments, please inform John
Aalegaard immediately and delete this e-mail and/or attachments.

**Reply by Lailloken** [reply]
Do you have to disable both, or is the (EDIT: external software) enough? I implemented the toggle for a single person who had issues almost 2.5 years ago, so I totally forgot to update it when I recently coupled omni-key item-features to the inventory pixel-check.
Could you check if disabling WindowTop is enough? I can fix item-related features for the next release.
***
PoE supports copying an item's text into clipboard, so the tool is not screen-reading it.

**Reply by S1lverone** [reply]
I see I never replied to this. I'm sorry for that. Let us continue in the other chat I just opened. I will not be playing poe1 more since ggg poe1 engine or what ever will never be fixed tonrun endgame builds without lag for me.
